src/components/layouts/AddressOverview.vue

```diff
@@ -219,6 +217,21 @@ export default defineComponent({
         const unclaimedCashlinkCount = ref(0);
         const showUnclaimedCashlinkList = ref(false);
 
+        onMounted(() => {
+            // Using setTimeout just to make sure, this is executed at the end of stack call as sometimes, it adds the
+            // 'masked' class although it's not needed
+            setTimeout(() => maskAddress(), 1);
+
+            window.addEventListener('resize', maskAddress);
+        });
+        onUnmounted(() => window.removeEventListener('resize', maskAddress));
+
+        function maskAddress() {
+            const addressElement = context.refs.$address as HTMLDivElement;
```

Comment:
The way to define template refs in the Composition API is with an actual `ref()`:

```js
const $address = ref<HTMLDivElement>(null);
```

Then this becomes:
```suggestion
            const addressElement = $address.value!;
```